src/data/auction.ts

```diff
@@ -55,7 +58,7 @@ export function generateAuctionData(unixTimestamp: number): Auction[] {
     {
       id: 'simulation',
       startBlock: addHours(dateUTC, -55).unix(),
-      endBlock: addHours(dateUTC, +2).unix(),
+      endBlock: addMinutes(dateUTC, +1).unix(),
```

Comment:
Can we think of a case where the investor wants to observe the IDO before they decide to place a bid?

Reply:
I'll keep that in mind. Although this ties into martin's other issue about making the bids more realistic. Should we include this point at that issue as well?